### PR TITLE
Assign roles

### LINKED ISSUE #42: Create role for person
Role describes the job description of the person. Role should be an Enum that is assigned to person.

### LINKED ISSUE #43: Assign role to person
Add method to assign role to person

### PR DESCRIPTION
**Pull request by karthickkc**
Added functionality to assign roles for each contact

### CODE REVIEW

**Review by bbbbcccd — COMMENTED** [review]
Fix check style and javadocs

**Comment by karthickkc** [comment]
fixes #43

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -42,6 +44,7 @@
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
+//import seedu.address.model.person.Role;
```
Comment (wes-w-z-h) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java]: dead code should be removed

**Review by wes-w-z-h — CHANGES_REQUESTED** [review]
lgtm minor changes

**Review by wes-w-z-h — APPROVED** [review]
lgtm!! 👍👍👍

**Pull request merged**